todo_app/data/trello_items.py

```diff
@@ -0,0 +1,86 @@
+from flask import session
+from ..classes.item_class import Item
+import json
+import os
+import requests
+import uuid
+
+"""
+    IMPORTANT - Due to work environment and working localhost, requests api causes SSL error
+    Therefore verify=False is necessary on requests call as a work-around
+"""
+
+trello_boards_url       = "https://api.trello.com/1/boards/"
+trello_cards_url        = "https://api.trello.com/1/cards"
+trello_board_id         = os.environ.get('TRELLO_BOARD_ID','')
+trello_board_list_id_todo = os.environ.get('TRELLO_BOARD_LIST_ID_TODO','')
+trello_board_list_id_done = os.environ.get('TRELLO_BOARD_LIST_ID_DONE','')
+trello_api_key          = os.environ.get('TRELLO_API_KEY','')
+trello_api_token        = os.environ.get('TRELLO_API_TOKEN','')
+trello_api_security     = "key="+trello_api_key+"&token="+trello_api_token
+trello_default_get_header   = {"Accept": "*/*"}
+trello_default_put_header   = {"Accept": "application/json"}
+trello_default_post_header   = {"Accept": "application/json"}
+cards_status_options    = ['all','closed','none','open','visible']
+
+def get_items():
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+    get_item_req_url = trello_boards_url+trello_board_id+'/cards/?'+trello_api_security 
+    response = requests.request("GET", get_item_req_url, data='',  headers=trello_default_get_header, verify=False)
+
+    response_json = response.json()
+    
+    todo_items = []
+    done_items = []
+
+    for item in response_json:
+        if item.get('idList','') == trello_board_list_id_todo:
+            todo_items.append(Item(
+                    item.get('id',''),
+                    item.get('name',''),
+                    'todo'
+                ))
+        elif item.get('idList','') == trello_board_list_id_done:
+            done_items.append(Item(
+                    item.get('id',''),
+                    item.get('name',''),
+                    'done'
+                ))
+
+    return_items = {'todo':todo_items,'done':done_items}
+    session.clear()
+    return session.get('items', return_items.copy())
```

Comment:
Given that you clear the session right before this, what is the point of asking for a key, won't it always just default to the second value? So is there any advantage over abandoning sessions, and using:
```suggestion
    return return_items
```